ha/setup/ha_setup.py

```diff
@@ -855,6 +855,16 @@ def main(argv: list):
             log_path = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}path")
             log_level = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}level")
             Log.init(service_name='ha_setup', log_path=log_path, level=log_level)
+        elif sys.argv[1] == "cleanup":
+            if not os.path.exists('const.HA_CONFIG_FILE'):
+                a_str = f'Cleanup can not be proceed as \
+                           HA config file: {const.HA_CONFIG_FILE} \
+                           is missing. Either cleanp is already done or there \
+                           is some other problem'
+                Log.error(f'{a_str}')
+                sys.stderr.write(a_str)
+                return
+            ConfigManager.init("ha_setup")
```

Comment:
Why ConfigManager.init() here?

Reply:
Don't we require init for cleanup where ha_conf file is not missing ?
Its the part when option is cleanup and conf file is present.

Reply:
Not clear on this.... If the option is cleanup and when the ha.conf file is  present, why do we need the init() ?? 
We are nyways going to remove everything right ?
Won't the next "config" CLI when issued, take care of the init() ?

Reply:
Actually this was already their earlier. For cleanup also, init was executing.
So, ConfigManager init() performs two things:
1. Load HA conf file.
2. Initializes the logs if not already.
Let me check the cleanup without init. How that works and will get back to you on this !

Reply:
I am keeping this init part here as cleanup logs will be logged and logging initialization will happen here. If you still have any concerns, let me know. And otherwise also, this part was already present earlier also.